quranref/scripts/.env

```diff
@@ -0,0 +1,7 @@
+DB_HOST=127.0.0.1
+DB_PORT=8529
+DB_NAME=quranref
+DB_USERNAME=kashif
+DB_PASSWORD=compulife
+DB_COLLECTION=ayas
```

Comment:
This is not required here as this is not a configuration setting. Wherever you create the DB, the collection names are still going to be the same.